pkg/plugins/utilities.go

```diff
@@ -135,16 +136,77 @@ func RefreshPluginManifest(ctx context.Context, config config.IConfig, fs afero.
 	return nil
 }
 
-func validatePluginManifest(body []byte) error {
+func fetchAndMergeManifests(pluginData requests.PluginData) (*PluginList, error) {
+	pluginList, err := fetchPluginList(pluginData.PluginBaseURL, "plugins.toml")
+	if err != nil {
+		return nil, err
+	}
+
+	additionalPluginLists := []*PluginList{}
+	for _, filename := range pluginData.AdditionalManifests {
+		additionalPluginList, err := fetchPluginList(pluginData.PluginBaseURL, filename)
+		if err != nil {
+			return nil, err
+		}
+		additionalPluginLists = append(additionalPluginLists, additionalPluginList)
```

Comment:
do we keep appending every time install command is ran? or do we only merge if not exist

Reply:
I want to ensure the user's plugins.toml file stays up to date, so I made this fetch and merge each file every time the install command is run.